lib/oplog/tail.go

```diff
@@ -143,56 +172,137 @@ func (tailer *Tailer) tailOnce(out chan<- *redispub.Publication, stop <-chan boo
 
 		var rawData bson.Raw
```

Comment:
bring down to after line 178?

Reply:
Yeah, I think so - I want to be clear that this isn't reused between loops..